Sources/hostmgr/helpers/Logging.swift

```diff
@@ -1,14 +1,38 @@
 import Foundation
 import Logging
 
-var logger = Logger(label: "com.automattic.hostmgr")
-
-func initializeLoggingSystem() {
-    let logLevelFromEnv = ProcessInfo.processInfo.environment["LOG_LEVEL"].flatMap { Logger.Level(rawValue: $0) }
-    logger.logLevel = logLevelFromEnv ?? .info
-    LoggingSystem.bootstrap { label in
-        MultiplexLogHandler([
-            StreamLogHandler.standardError(label: label)
-        ])
+// swiftlint:disable type_name
```

Comment:
Maybe enable this rule after the struct declaration?

Reply:
Ah good to have just in case!

Addressed in e5ed7b1f8b5510c80c48670f25167dc225ab7dc3